pkg/occlient/occlient_test.go

```diff
@@ -916,9 +916,9 @@ func TestSetupForSupervisor(t *testing.T) {
 					t.Errorf("created PVC spec not matching with expected values, expected: %s, got %s", createdPVC.Spec, tt.createdPVC.Spec)
 				}
 			} else if err == nil && tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "error", "no error")
+				t.Error("test failed, expected: error, got no error")
```

Comment:
When you see just this error it is not clear what happened.
"error was expected, but no error returned" or something like this